### ISSUE TITLE
Is a workflow necessary?

### ISSUE THREAD

**Issue by bronger** [ResearchObject/ro-crate-validator-py#2]
You seem to be able to leave the function https://github.com/ResearchObject/ro-crate-validator-py/blob/c3c5c18509c00ff51a96d430ed5ba5dc3e50b86c/src/rocrateValidator/semanticCheck.py#L769-L816
only if you have at least one workflow. But is such a thing mandatory in an RO-Crate?

**Comment by stain** [comment]
You are right, this check should rather be disabled unless the crate claims it is following the more specific [Workflow RO-Crate](https://about.workflowhub.eu/Workflow-RO-Crate/) profile.
